src/test/java/mailYandexPage/HomePage.java

```diff
@@ -25,6 +28,8 @@ public class HomePage {
     public HomePage(WebDriver driver) {
         this.driver = driver;
         PageFactory.initElements(driver, this);
+        ScreenshotUtil.getSnapShot(driver, "src/test/java/screenshots/" + HomePage.class.getSimpleName() + " "
```

Comment:
Unusual place to do screenshot as part as constructor initialization :) 
Better to do after some step

Reply:
fixed